justin-cotarla/big-wiki-crew
# Feature/view notes from article

Associated to #76 , viewing notes from a Wiki article.

**This PR includes**:
- Complete layout to view notes from an article
- Basic functionality to view notes from an article (scrollable notes, note copying on long press..)
- Handling empty notes edge case
- UI test

**Missing**:
- Integration with notes API to retrieve actual notes (currently hardcoded a short list as an example)

**To test**:
1. View a Wikipedia article.
2. Click on the new notes icon in the bottom action tab.
3. Interact with notes viewer dialog
3.1. Scroll through the dialog horizontal to see more notes.
3.2. Scroll through card view vertically to see longer notes.
3.3. Long press on card view to interact with context menu

**Example view**:
<img src="https://user-images.githubusercontent.com/23409994/55758318-66b57000-5a24-11e9-81f2-d1c9ed663257.jpg" width="300"/>
